apps/measures/models.py

```diff
@@ -139,7 +139,27 @@ class MeasuresOverviewPage(MetadataPageMixin, Page):
         ObjectList(de_ls_content_panels, heading='Easy German'),
     ])
 
-    subpage_types = ['MeasuresDetailPage']
+    subpage_types = ['apps_measures.MeasuresDetailPage']
+
+    @property
+    def status(self):
+        return dict(STATUS_CHOICES)
+
+    @property
+    def district(self):
+        return dict(BEZIRK_CHOICES)
+
+    @property
+    def measures_pages(self):
+        measures_pages = MeasuresDetailPage.objects.live().descendant_of(self)
+
+        return measures_pages
+
+    def get_context(self, request):
+        measures_pages = self.measures_pages
+        context = super().get_context(request)
+        context['measures_pages'] = measures_pages
+        return context
```

Comment:
@Rineee @fuzzylogic2000 I just copied this context from liqd and removed what I thought we didn't need but might be clunky, also I assume we will need additional stuff for the filtering?